api/src/backend/api/v1/views.py

```diff
@@ -1150,6 +1152,28 @@ def destroy(self, request, *args, pk=None, **kwargs):
             404: OpenApiResponse(description="The scan has no reports"),
         },
     ),
+    compliance=extend_schema(
+        tags=["Scan"],
+        summary="Retrieve compliance report as CSV",
+        description="Download a specific compliance report (e.g., 'cis_1.4_aws') as a CSV file.",
+        operation_id="scan_compliance_download",
```

Comment:
```python
operation_id="scan_compliance_download",
```

Are we also adding this field from now on? To keep it in mind. I believe it just serves a purpose for documentation.

Reply:
For the moment I remove it and if we decide to add it we will do it at the same time in a PR